ansible/roles/microshift-start/tasks/main.yml

```diff
@@ -1,21 +1,48 @@
 ---
-# install-microshift tasks
+# microshift-start tasks
+
+- name: cleanup microshift data
+  ansible.builtin.shell: echo 1 | /usr/bin/cleanup-all-microshift-data
+  when: cleanup_microshift | default('true') | bool
+
+- name: disable microshift service
+  become: yes
+  ansible.builtin.systemd:
+    name: microshift
+    state: stopped
+    enabled: no
+
+- name: create .kube home dir
+  ansible.builtin.file:
+    path: ~/.kube/
+    state: directory
 
 - name: reboot machine
-  reboot:
+  become: yes
+  ansible.builtin.reboot:
     reboot_timeout: 600
 
+- name: record post-install disk utilization
+  vars:
+    filename: disk1.txt
+  include_tasks: roles/common/tasks/disk.yml
+  loop: "{{ du_dirs }}"
+
 - name: Pause for 60 seconds for steady state post reboot
   ansible.builtin.pause:
     seconds: 60
 
 - name: pbench recording start
+  become: yes
   block:
   - name: source pbench-agent & register-tool-set
     ansible.builtin.shell: source /etc/profile.d/pbench-agent.sh && pbench-register-tool-set
   
+  - name: set new pidstat interval
+    ansible.builtin.shell: source /etc/profile.d/pbench-agent.sh && pbench-register-tool --name=pidstat -- --interval={{ sample_interval }}
+
   - name: start pbench-user-benchmark recording
-    ansible.builtin.shell: source /etc/profile.d/pbench-agent.sh && pbench-user-benchmark --config=microshift2 -- sleep {{ pbench_record_duration }}
+    ansible.builtin.shell: source /etc/profile.d/pbench-agent.sh && pbench-user-benchmark --config=microshift -- sleep {{ pbench_record_duration }}
```

Comment:
Don't we need to add condition here for pbench presence? We may want to collect stats using Prometheus while pbench is not installed.

Reply:
Again, we may also wait to refactor this start/stop code to a separate task in commons to keep this file managable

Reply:
Line 35 to Line 55 is a logical block (L37), the entire block of tasks to "start bench-user-benchmark recording" is conditional on `install_pbench` being true (L53).